app/services/katello/candlepin_event_listener.rb

```diff
@@ -1,47 +1,24 @@
 module Katello
   class CandlepinEventListener
-    PROCESSED_COUNT_CACHE_KEY = 'candlepin_events_processed'.freeze
-    FAILED_COUNT_CACHE_KEY = 'candlepin_events_failed'.freeze
-    AMQP_QUEUE_NAME = 'event.org.candlepin.audit.AMQPBusPublisher'.freeze
+    STATUS_CACHE_KEY = 'candlepin_events_status'.freeze
 
     CandlepinEvent = Struct.new(:message_id, :subject, :content)
 
     @logger = ::Foreman::Logging.logger('katello/candlepin_events')
     @failed_count = 0
     @processed_count = 0
 
-    def self.start_service
```

Comment:
Since the Monitor will try to restart the service if it loses connection (`status[:running]` would be false) this recovery code is totally redundant